specifications/catalog/example/dcat.distribution.example.json

```diff
@@ -0,0 +1,84 @@
+{
+    "@context":  "https://w3id.org/idsa/v5/context.json",
+    "@id": "http://provider.com/catalog1/dataset1/",
+    "@type": "dcat:Dataset",
+    "dct:title" : "IDS Dataset #1",
+    "dct:description": [ {
+        "@language": "en",
+        "@value": "This is the number 1 data asset of the IDS. The consumer must be a TRUST_PLUS certfified connector."
+    }, {
+        "@language": "de",
+        "@value": "Dies ist das Nummer 1 Data Asset im IDS. Der Consumer muss ein TRUST_PLUS zertifizierter Connetor sein."
+    } ],
+    "dcat:keyword": [
+        "IDS",
+        "data asset",
+        "dcat dataset"
+    ],
+    "dcat:distribution": [ {
+        "@type": "dcat:Distribution",
+        "dct:title" : "Distribution#1",
+        "dct:description": "This is a distribution which provides the data via a HTTP_REST dataplane.",
+        "dcat:keyword": [
+            "dataplane",
```

Comment:
maybe just "HTTP" and "REST" as keywords? Actually, do we even need keywords here?

Reply:
Not really, just want to be "sufficiently" complete.